frontend/src/app/main/ui/shapes/attrs.cljs

```diff
@@ -120,6 +120,17 @@
         (obj/merge! attrs (clj->js stroke-attrs)))
       attrs)))
 
+(defn add-layer-props [attrs shape]
+  (let [layer-attrs
+        (cond-> {}
```

Comment:
maybe here a combination of `obj/merge!` + `when` will result in a more efficient code

Reply:
or better: (obj/new) + `cond->`